### PR TITLE
docs: footer

### PR DESCRIPTION
**Pull request by abbeyperini-deque**
Marketing request for Hacktoberfest: A footer, with link back to [deque.com](http://deque.com/) and a link to the open-source terms surrounding this project.

- create component with footer tag
- add copyright, link to deque.com, link to terms of use, and link to MPL-2.0 License
- position sidebar and layout so they're not blocked by the footer

Issue #739

### CODE REVIEW

File: docs/Demo/Footer.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from '@deque/cauldron-react/';
+
+const Footer = props => {
+  return (
+    <footer className="Footer">
+      <ul className="Footer-links">
+        <li>
+          Copyright © 2022{' '}
```
Comment (scurker) [inline docs/Demo/Footer.js:9]: Should we not calculate the year? i.e.

```tsx
const year = new Date().getFullYear()
return (
   ...
   <li>
     Copyright © {year}
   </li>
)
```

File: docs/index.css [hunk 2]
```diff
@@ -46,3 +46,35 @@ code {
 .Demo ul {
   margin: var(--space-large);
 }
+
+.Footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: var(--top-bar-height);
```
Comment (scurker) [inline docs/index.css:54]: I'm not sure we need the footer visible at all times. Can it not just be a footer inside of the workspace section?

**Review by scurker — CHANGES_REQUESTED** [review]

File: docs/index.css (hunk 2 above)
Comment (abbeyperini-deque) [reply docs/index.css:54]: `<footer>` can't go inside `<main>`, so I can't make it a child of `<Workspace>`. I can style it that way, but I thought it looked weird when I tried it initially.

Full width:
Screen Shot 2022-08-23 at 10 25 11 AM

Workspace width:
Screen Shot 2022-08-23 at 10 35 15 AM

File: docs/Demo/Footer.js (hunk 1 above)
Comment (abbeyperini-deque) [reply docs/Demo/Footer.js]: These changes have been pushed!

File: docs/index.css (hunk 2 above)
Comment (karnett) [reply docs/index.css:54]: eek lol - I like it full width personally

**Pull request merged**